projects/VitalRecord/VitalRecord_util.cs

```diff
@@ -1151,6 +1151,22 @@ protected void SetCodeValue(string field, string code, string[,] options)
             throw new System.ArgumentException($"Code '{code}' is not an allowed value for field {field}");
         }
 
+        /// <summary>Helper function to determine whether a value appears in the the set of allowed codes.</summary>
+        // <param name="code">the code to check.</param>
+        // <param name="options">the list of valid options and related display strings and code systems</param>
+        protected bool CodeExistsInValueSet(string code, string[,] options)
```

Comment:
This validator ties into the discussion we had with Rob about validation. If a provided code doesn't exist, and we will do nothing, we should add a message to the validationErrors array or log a warning saying the code was unrecognized and will be ignored

Reply:
Added a log since validationErrors is stored on the IJE side.